interactions/models/discord/auto_mod.py

```diff
@@ -175,6 +188,14 @@ class TimeoutUser(BaseAction):
     type: AutoModAction = attrs.field(repr=False, default=AutoModAction.TIMEOUT_USER, converter=AutoModAction)
 
 
+@attrs.define(eq=False, order=False, hash=False, kw_only=False)
+class BlockMemberInteraction(BaseAction):
+    """Block a member from using text, voice, or other interactions"""
+
+    # this action has no metadata
+    ...
```

Comment:
You don't need this line (3 dots) because you have docstring

Reply:
Surprised you didn't see the other redundant ones 

resolved